pkg/flags/strings.go

```diff
@@ -22,6 +22,20 @@ func NewStringsFlag(valids ...string) *StringsFlag {
 	return &StringsFlag{Values: valids}
```

Comment:
instead of having a new function, maybe use the first element of the `valids` slice as the default here?
```go
return &StringsFlag{Values: valids, val: valids[0]}
```